### DISCUSSION TITLE
Improve User Interface/User Experience

### DISCUSSION THREAD

**Discussion by Krylan** [RetroAchievements/RAWeb#767] [Ideas]
From the first sight, website looks a little messy, It's also not fully responsive and not adapted to newest web development standards. I'm aware of the fact, that "revolutions" in product design are often on the beginning received badly by users. That's why, in my opinion, it would be better to make smaller changes to the existing layout.
I would like to suggest doing few changes in order to improve User Interface/User Experience on Retro Achievements:
– introduce RWD/AWD (Responsive/Adaptive Web Design) to make website more responsive (using media query in CSS), not depending on detecting mobile device
– simplify some elements' styles of the website, for example making footer or navigation more readable
– correct mistakes (for example: when right container is taller than left one, it covers the footer)
– improve website semantics to suit modern standards of HTML5 and SEO

If you would agree on that, I can start working on it.

**Comment by luchaos** [comment]
i answered a similar question here https://github.com/RetroAchievements/RAWeb/issues/42#issuecomment-477107414
feel free to do it for v1 but i cannot guarantee that i'll incorporate those changes into v2 as already a lot of work has gone into it

**Comment by jeffjenx** [comment]
If you need any help with this stuff, I've maintained a career in front-end web development for nearly two decades now. If it's providing some guidance and direction, or peer programming, or help building out a proper design system, I've got a lot of experience on projects from small to large to very large.

**Comment by kdecks** [comment]
That's awesome! I hope we can help you to get involved.
